azkaban-common/src/main/java/azkaban/project/FlowLoaderUtils.java

```diff
@@ -148,6 +232,33 @@ public static ValidationReport generateFlowLoaderReport(final Set<String> errors
     return report;
   }
 
+  /**
+   * Clean up temp dir.
+   *
+   * @param tempDir the temp dir
+   */
+  public static void cleanUpTempDir(final File tempDir) {
```

Comment:
might just call it cleanupdir since the method can be applied to any kind of  dir deletion.

Reply:
Sure.